rootfs-builder/rootfs.sh

```diff
@@ -381,6 +381,13 @@ echo "refclock PHC /dev/ptp0 poll 3 dpoll -2 offset 0" >> ${chrony_conf_file}
 # Reference:  https://chrony.tuxfamily.org/doc/3.4/chrony.conf.html
 sed -i 's/^\(server \|pool \|peer \)/# &/g'  ${chrony_conf_file}
 
+chrony_systemd_service="${ROOTFS_DIR}/usr/lib/systemd/system/chronyd.service"
+if [ ${distro} == ubuntu ] || [ ${distro} == debian ] ; then
+	chrony_systemd_service="${ROOTFS_DIR}/lib/systemd/system/chrony.service"
+fi 
+
+sed -i 's/^Unit/&ConditionPathExists=/dev/ptp0/' ${chrony_systemd_service}
```

Comment:
Is this going to work as you have slashes in the replacement text? Maybe:

```sh
sed -i 's!^Unit!&ConditionPathExists=/dev/ptp0!' ${chrony_systemd_service}
```

Reply:
@jodh-intel Looks like I missed pushing my latest changes. I have addressed this. PTAL.